executor_manager/routers/routers.py

```
@@ -22,7 +22,9 @@
 from executor_manager.clients.task_api_client import TaskApiClient
 from shared.models.task import TasksRequest
 from executor_manager.executors.dispatcher import ExecutorDispatcher
-from typing import Optional, Dict, Any
+from executor_manager.services.restart_limiter import get_restart_limiter
+from executor_manager.executors.docker.utils import delete_container
+from typing import Optional, Dict, Any, List
```

Ruff (0.14.6)

Redefinition of unused `Optional` from line 18

Remove definition: `Optional`

(F811)